modules/social_features/social_core/social_core.install

```diff
@@ -1639,3 +1639,17 @@ function social_core_update_11404(): void {
 
   $role->setWeight(3)->save();
 }
+
+/**
+ * Update social_post exposed filter button name.
+ */
+function social_core_update_11405(): string {
+  /** @var \Drupal\update_helper\UpdaterInterface $update_helper */
```

Comment:
Will it be a good idea to check if `social_post_update_11401` has run or not? I mean, to run the update only if the schema of social_post module is not 11405.

Reply:
We could yes, but I thought it wouldn't really matter if it would run twice as the end result would still be the same.

Reply:
Check, okay.